packages/sui-studio/bin/helpers/copyStaticFiles.js

```diff
@@ -0,0 +1,18 @@
+const cpy = require('cpy')
+
+module.exports = function copyStaticFiles() {
```

Comment:
Instead bundling the static files, we're just copying them and later just fetching the files. So it doesn't depend on webpack anymore.

Reply:
😮 Good one!!